cdc/sink/metrics.go

```diff
@@ -34,6 +34,14 @@ var (
 			Help:      "Bucketed histogram of processing time (s) of a txn.",
 			Buckets:   prometheus.ExponentialBuckets(0.002 /* 2 ms */, 2, 18),
 		}, []string{"capture", "changefeed"})
+	execDDLHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: "ticdc",
+			Subsystem: "sink",
+			Name:      "ddl_exec_duration",
+			Help:      "Bucketed histogram of processing time (s) of a ddl.",
+			Buckets:   prometheus.ExponentialBuckets(0.002, 2, 18),
```

Comment:
```suggestion
			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 18),
```

For DDL events, we cares long running events, so I think we can range from 10ms to 43min